integrations/ollama/pyproject.toml

```
@@ -0,0 +1,85 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "ollama-haystack"
+dynamic = ["version"]
+description = 'An integration between the Ollama LLM framework and Haystack'
+readme = "README.md"
+requires-python = ">=3.8"
+license = "Apache-2.0"
+keywords = []
+authors = [
+  { name = "Alistair Rogers", email = "alistairlr112@gmail.com" },
+  { name = "Sachin Sachdeva", email = "emailforsachinsachdeva@gmail.com" }
+]
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+dependencies = ["haystack-ai", "requests"]
+
+[project.urls]
+Documentation = "https://github.com/unknown/ollama-haystack#readme"
+Issues = "https://github.com/unknown/ollama-haystack/issues"
+Source = "https://github.com/unknown/ollama-haystack"
+
+[tool.hatch.version]
+path = "src/ollama_haystack/__about__.py"
```

```suggestion
[tool.hatch.version]
source = "vcs"
tag-pattern = 'integrations\/ollama-v(?P<version>.*)'

[tool.hatch.version.raw-options]
root = "../.."
git_describe_command = 'git describe --tags --match="integrations/ollama-v[0-9]*"'
```

Lately, we are using this different strategy for versioning.